lib/shoryuken/processor.rb

```diff
@@ -22,8 +22,7 @@ def process
         end
       end
     rescue Exception => ex
-      logger.error { "Processor failed: #{ex.message}" }
-      logger.error { ex.backtrace.join("\n") } unless ex.backtrace.nil?
+      Array(Shoryuken.exception_handlers).each { |handler| handler.call(ex, queue, sqs_msg) }
```

Comment:
Is there a way you can test this too? Maybe here https://github.com/ruby-shoryuken/shoryuken/blob/main/spec/shoryuken/processor_spec.rb

Reply:
This actually is tested in `spec/shoryuken/default_exception_handler_spec.rb` with InlineExecutor, but I can add more tests

Reply:
nvm I misread that. It looks great! Thank you again.